citadel/hotkeys.md

```diff
@@ -0,0 +1,39 @@
+# Ignition Keyboard Shortcuts
+
+<div style="display: table">
+
+
+
+  <table cellspacing='0'> <!-- cellspacing='0' is important, must stay -->
+
+  <tr><th>Action</th><th>Operation</th><th>Instruction</th></tr><!-- Table Header -->
+
+  <tr><th colspan="3" class="sub"><b>MODE</b></th></tr>
+
+  <tr><td>Enter Selection mode (default mode)</td><td>Esc</td><td>press Esc for mode to select models </td></tr>
```

Comment:
Maybe update this description to parallel the two below rows for rotation and translation? Same for blueprint